CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 3.2.0 (2020-01-06)
+## 3.3.0 (2020-03-24)
+
+- Bump acorn from 6.3.0 to 6.4.1 to address security vulnerability. [#56](https://github.com/blackbaud/skyux-lib-help/pull/56)
```

Comment:
Based on how they punctuate it at https://github.com/acornjs/acorn, I'm pretty sure `Acorn` should be capitalized.

Also, we generally make version numbers code snippets in the changelogs. I cheched the `skyux-lib-help` changelog, and we only have one such instance (on 2019-06-21), and it is a code snippet. So I'd make `6.3.0` and `6.4.1` code snippets.

I'd also change `Bump` to `Updated` because our style in changelogs is to use past tense verbs and because we stick to a limited number of those (Added/Updated/Fixed/Deleted). And I'd add the missing `a` before `security`.

I'll go ahead and make all of these changes, but I'm noting them in this comment in case you disagree with any of them and want to discuss further or just change them back.